### PR TITLE
dev-python/imapclient: Bump to 2.0.0

### PR DESCRIPTION
**Pull request by luke-jr**
Adds keyword ~ppc64 also, as that is my system where I test.

Closes: https://bugs.gentoo.org/571310

Note: Tests require an IMAP server, so can't very well be supported.

### CODE REVIEW

File: dev-python/imapclient/imapclient-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (sbraz) [inline dev-python/imapclient/imapclient-2.0.0.ebuild]: Could you do EAPI=7?

File: dev-python/imapclient/imapclient-2.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} pypy )
```
Comment (sbraz) [inline dev-python/imapclient/imapclient-2.0.0.ebuild]: Can you try to add pypy3 if it's supported?

File: dev-python/imapclient/imapclient-2.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+inherit distutils-r1
+
+MY_PN="IMAPClient"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="easy-to-use, pythonic, and complete IMAP client library"
+HOMEPAGE="http://imapclient.freshfoo.com/"
```
Comment (sbraz) [inline dev-python/imapclient/imapclient-2.0.0.ebuild]: This seems to redirect to https://github.com/mjs/imapclient

File: dev-python/imapclient/imapclient-2.0.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="doc test"
+
+RDEPEND="dev-python/six[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
```
Comment (sbraz) [inline dev-python/imapclient/imapclient-2.0.0.ebuild]: I doubt that you need RDEPEND in DEPEND

File: dev-python/imapclient/imapclient-2.0.0.ebuild (hunk 2 above)
Comment (luke-jr) [reply dev-python/imapclient/imapclient-2.0.0.ebuild]: Can't test. pypy3 isn't supported for my arch (ppc64).

File: dev-python/imapclient/imapclient-2.0.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? ( $(python_gen_cond_dep '>=dev-python/mock-1.3.0[${PYTHON_USEDEP}]' python2_7 pypy) )"
```
Comment (mgorny) [inline dev-python/imapclient/imapclient-2.0.0.ebuild]: Optionally: you can use `-2` to indicate all versions of Python 2.

File: dev-python/imapclient/imapclient-2.0.0.ebuild (hunk 4 above)
Comment (mgorny) [reply dev-python/imapclient/imapclient-2.0.0.ebuild]: Yes, AFAICS it's needed only in `test?`.

File: dev-python/imapclient/imapclient-2.0.0.ebuild (hunk 2 above)
Comment (mgorny) [reply dev-python/imapclient/imapclient-2.0.0.ebuild]: That's fine.

File: dev-python/imapclient/imapclient-2.0.0.ebuild (hunk 3 above)
Comment (mgorny) [reply dev-python/imapclient/imapclient-2.0.0.ebuild]: FWICS pypi page also uses the GitHub page as homepage. Also, please append the pypi homepage here as well.

File: dev-python/imapclient/imapclient-2.0.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} pypy )
+
+inherit distutils-r1
+
+MY_PN="IMAPClient"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="easy-to-use, pythonic, and complete IMAP client library"
```
Comment (mgorny) [inline dev-python/imapclient/imapclient-2.0.0.ebuild]: Capitalize 'easy', please.

**Comment by mgorny** [comment]
Ping.

**Comment by luke-jr** [comment]
Not likely to have time to address this in at least the next month. :(

**Comment by mgorny** [comment]
Ping².

**Pull request closed without merging**